O365-iOS-Microsoft-Graph-Connect-Swift/AuthenticationClass.swift

```diff
@@ -50,27 +53,29 @@ class AuthenticationClass {
                                                _ accessToken: String) -> Void) {
         do {
             if let initError = self.lastInitError {
-                throw NSError.init(domain: initError, code: 0, userInfo: nil)
+                throw NSError(domain: initError, code: 0, userInfo: nil)
```

Comment:
why is this one necessary? Specifically with code:0?

Reply:
I didn't change much on the original logic. But here seems very weird. I have changed to throw the original init error which will do the job.